app/controllers/api/v1/keywords_controller.rb

```diff
@@ -31,6 +34,20 @@ def show
 
       private
 
+      def matching_adswords_urls
+        keywords_query = KeywordsQuery.new(Keyword, filter_params)
+        pagination, keywords = paginated_authorized(keywords_query.call)
+        keyword_presenters = keywords.map { |item| KeywordPresenter.new(item, filter_params) }
+
+        render json: KeywordSerializer.new(keyword_presenters, meta: meta_from_pagination(pagination))
+      end
+
+      def all_keywords
+        pagination, keywords = paginated_authorized(Keyword)
+
+        render json: KeywordSerializer.new(keywords, meta: meta_from_pagination(pagination))
+      end
```

Comment:
```suggestion
      def filtered_keywords
        keywords_query = KeywordsQuery.new(Keyword, filter_params)
        pagination, keywords = paginated_authorized(keywords_query.call)
        keyword_presenters = keywords.map { |item| KeywordPresenter.new(item, filter_params) }

        render json: KeywordSerializer.new(keyword_presenters, meta: meta_from_pagination(pagination))
      end
```

- rename the method to describe what it does without assuming what the "filters" are (later we might add more filters, it's the responsibility of the query object to care about what filter can be used).
- remove the `all_keyworsr` method: the query object must return all keywords if there is no filter provided.